packages/cli/src/commonlib/log.ts

```diff
@@ -36,6 +36,10 @@ export class CLILogProvider implements LogProvider {
     return CLILogProvider.instance;
   }
 
+  getLogFilePath(): string {
+    throw new NotImplementedError("CLILogProvider", "getLogFilePath");
```

Comment:
Does this mean CLI has different validation behavior from VSC?

Reply:
Same worry about the server code (VS).

Reply:
CLI will output all the validation logs directly, won't output to log file. As the log folder path it's hard to generate, needs to consider different OSs, but it may be supported in future versions.
VS is the same, I will investigate if it's possible to get default extension log folder.

Reply:
OK to me if that's a known point or as designed.

Reply:
As discussed with @Alive-Fish , will provide the log folder under %TEMP% for CLI/server to avoid potential error thrown.

Reply:
Is that a hard-coded file name or with timestamp prefix/suffix?

Reply:
E.g. log file:
"C:\Users\xx\AppData\Local\Temp\.fx\server-log\20230330T030429.log"